library/src/commonMain/kotlin/com/lagradost/cloudstream3/MainAPI.kt

```diff
@@ -681,8 +681,11 @@ enum class TvType(value: Int?) {
     Music(13),
     AudioBook(14),
 
-    /** Wont load the built in player, make your own interaction */
+    /** Won't load the built in player, make your own interaction */
     CustomMedia(15),
+
+    Audio(16),
```

Comment:
We have music and audiobook, what exactly is Audio?

Reply:
Something that doesnt cover any of it. My main reason was for https://github.com/Luna712/Luna712-CloudStream-Extensions/blob/6f45efaa365c9cf3a65f73a848d5cf2e10820fbd/InternetArchiveProvider/src/main/kotlin/com/luna712/InternetArchiveProvider.kt#L256-L257 where no other type other than a generic Audio actually makes true sense there. It isn't absolutely necessary by any means though so I suppose I can take it out if you really want?